Tests/XCRemoteCacheTests/Network/NetworkClientImplTests.swift

```diff
@@ -208,7 +208,7 @@ class NetworkClientImplTests: XCTestCase {
             service: "iam",
             date: Date(timeIntervalSince1970: 1_440_938_160)
         )
-        client = NetworkClientImpl(session: session, retries: 0, fileManager: fileManager, awsV4Signature: signature)
+        client = NetworkClientImpl(session: session, retries: 0, retryDelay: 0.1, fileManager: fileManager, awsV4Signature: signature)
         responses[url] = .success(successResponse, Data())
         _ = try waitForResponse { client.fetch(url, completion: $0) }
```

Comment:
Same here: 0.1 delay vs 0.1  timeout might be a bit risky.
What do you think about setting `retryDelay` to `0`?

Reply:
Totally agree, fixed.